index.html

```diff
@@ -13,10 +13,12 @@ <h1>Awesome books</h1>
     </div>
     <ul id="bookList"></ul>
     
```

Comment:
- [ ] Please keep in mind that you should delete the extra blank lines and indentations to maintain your code clean and the website optimized. 😊
_Kindly note that we add blank lines only for separating two logical pieces of code or two large sections apart._
**You find about this tip and more in this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)**